roles/common/tasks/main.yml

```diff
@@ -39,6 +39,28 @@
     name: network
     state: restarted 
   when: set_node_routes and inventory_hostname != 'localhost'
+  
+- name: Remove servers from chrony.conf
+  replace:
+    path: /etc/chrony.conf
+    regexp: '^server.*iburst$'
+    replace: ''
+  when: multinetwork and inventory_hostname in groups.net2_all
+
+- name: Add net2 ntp servers to chrony config file
+  lineinfile:
+    path: /etc/chrony.conf
+    regexp: '^#\sPlease\sconsider'
```

Comment:
Is this line needed? I didn't think you needed to match this line as well? Just use insertafter on its own.

Reply:
That line wasn't needed  - removed regexp line